---
description: '詳細情報: C28163'
title: C28163
ms.date: 11/04/2016
ms.topic: reference
f1_keywords:
- C28163
helpviewer_keywords:
- C28163
ms.assetid: 24fecbde-1c96-4a45-82f7-9f47cfc0ef11
ms.openlocfilehash: 89f17f9ef9aaeb6b3dc0710536440ed2fcfbc982
ms.sourcegitcommit: d6af41e42699628c3e2e6063ec7b03931a49a098
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 12/11/2020
ms.locfileid: "97303428"
---
# <a name="c28163"></a>C28163

> 警告 C28163: 関数を try/except ブロック内から呼び出さないでください

この警告は、ブロックで囲まれていない型の関数がブロックで見つかった場合に報告され `try/except` `try/except` ます。 コード分析ツールによって、関数が呼び出された少なくとも1つのパスがブロック内に見つかりました `try/except` 。
